targets/drive/mobile/main.jsx

```diff
@@ -43,28 +44,48 @@ window.handleOpenURL = require('drive/mobile/lib/handleDeepLink').default(
   hashHistory
 )
 
+const getPreviousToken = async store => {
+  if (store.getState().mobile.settings.token) return store.getState().mobile.settings.token
+  else {
+    // transition from version 1.5.3 to the next one, we recover the token stored by cozy-client-js
+    const cozyClientStorage = new Storage()
+    const { token } = await cozyClientStorage.load('creds')
+    store.dispatch(setToken(token))
+    return token
+  }
+}
+
 const startApplication = async function(store, client) {
   configureReporter()
-  const { client: clientInfos } = store.getState().settings
-    const { token } = store.getState().mobile.settings
-  if (clientInfos && token) {
+
```

Comment:
There were some problems with the next part, so I re-organized it:

- if the server is not responding or we're offline, we don't revoke ourselves
- but we do revoke ourself if there is no token or client information
- the cozy bar is always started if we're logged in, but not if we're on the authentication screen